python/cuspatial/cuspatial/core/_column/geocolumn.py

```diff
@@ -179,6 +179,25 @@ def copy(self, deep=True):
         )
         return result
 
+    @property
+    def valid_count(self) -> int:
+        """
+        Returns the number of valid geometries stored in this GeoColumn.
+        Null support is implemented in GeoSeries and this is not expected
+        to ever return less than the length of the GeoColumn. This is
+        provided as a cudf shim layer.
+        """
+        return self._meta.input_types.valid_count
```

Comment:
I thought cuspatial supports null, by setting `input_types` to `Feature_Enum.NONE`?

Reply:
I don't know. In theory arrow's union column doesn't support null. So you may not be able to.

Reply:
That's correct, good job looking it up because I had to implement it this morning to find that union column doesn't allow null for `union_offsets` or `input_types`. I'll look at actually counting Feature_Enum(NONE)s instead of returning the nullcount of the above which will always be 0.